wp-content/themes/core/theme.json

```diff
@@ -95,10 +95,100 @@
         }
       ]
     },
+    "custom": {
+      "fontWeight": {
+        "regular": 400,
+        "bold": 700
+      }
+    },
     "layout": {
-      "contentSize": "80ch",
+      "contentSize": "680px",
```

Comment:
Not related to styleguide typography but updated to a px value here so blocks align appropriately. Will update this again later when spacing and layout styleguide aspects are implemented.